### PR TITLE
[Docs site] Mermaid diagrams

### PR DESCRIPTION
**Pull request by kodster28**
Adds https://mermaid-js.github.io/mermaid/#/ as a templating option to auto-generate diagrams from text snippets.

### CODE REVIEW

File: content/load-balancing/_partials/_health-check-diagram.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+<div class="mermaid">
+    flowchart LR
+      accTitle: Load balancing monitor flow
+      accDescr: Dynamic load balancing involves pools, origins, monitors, and health. checks
```
Comment (ranbel) [inline content/load-balancing/_partials/_health-check-diagram.md]:
```suggestion
      accDescr: Dynamic load balancing involves pools, origins, monitors, and health checks.
```

File: package.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
     "postbuild": "tsm bin/highlight.ts restore"
   },
   "dependencies": {
-    "fromnow": "3.0.1"
+    "fromnow": "3.0.1",
+    "mermaid": "^9.1.7"
```
Comment (ranbel) [inline package.json:17]: should the ^ be there?

File: content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md [hunk 3]
```diff
@@ -16,7 +16,12 @@ Connections to the origin will always be made using SSL/TLS, regardless of the s
 
 The certificate presented by the origin will be validated the same as with [Full (strict) mode](/ssl/origin-configuration/ssl-modes/full-strict/).
 
-![With an encryption mode of Strict (SSL-Only Origin Pull), all connections to the origin will always be made using SSL/TLS](/ssl/static/ssl-encryption-mode-full-strict.png)
+<div class="mermaid">
+flowchart LR
+    accTitle: Strict (SSL-Only Origin Pull) SSL/TLS Encryption
+    accDescr: With an encryption mode of Strict (SSL-Only Origin Pull), all connections to the origin will always be made using SSL/TLS.
+    A[Browser] <--Encrypted--> B((Cloudflare))<--Encrypted--> C[("Origin server &#9989;")]
```
Comment (ranbel) [inline content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md]: What does the green checkmark icon indicate in this diagram?

**Review by ranbel — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (kodster28) [reply package.json:17]: I think so. That was the default installation version.

File: content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md (hunk 3 above)
Comment (kodster28) [reply content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md]: Added a word to clarify. It means that their cert has been verified.

**Pull request merged**